### PR TITLE
Strike two. (#180/#181)

### PR DESCRIPTION
**Pull request by GuardedLolz**
### Prerequisites
* [X] I have read and understood the [contribution guidelines](CONTRIBUTING.md)
* [X] This pull request follows the code style of the project
* [X] I have tested this feature thoroughly WITH THE USE of the load-all-sounds starter
  * capture
* [X] This pull request is merging into **`dev`**
* [X] I have filled out this pull request template knowing if I fail to complete it the pull request will immediately become invalid and will be closed

### Changes Proposed in this Pull Request
* See #181.

...

### CODE REVIEW

File: core/assets/sounds/cues/builtToScaleDS/data.json [hunk 1]
```diff
@@ -43,11 +43,13 @@
         {
           "id": "builtToScaleDS/c",
           "beat": 4,
+		  "duration": 1,
```
Comment (chrislo27) [inline core/assets/sounds/cues/builtToScaleDS/data.json]: Indentation

File: core/assets/sounds/cues/builtToScaleDS/data.json [hunk 2]
```diff
@@ -43,11 +43,13 @@
...
           "semitone": 7
         },
 		{
           "id": "builtToScaleDS/c",
           "beat": 4,
+		  "duration": 1,
```
Comment (chrislo27) [inline core/assets/sounds/cues/builtToScaleDS/data.json]: Indentation

File: core/assets/sounds/cues/clappyTrio/data.json [hunk 3]
```diff
@@ -35,7 +35,8 @@
         },
         {
           "id": "clappyTrio/clap",
-          "beat": 2.0
+          "beat": 2.0,
+		  "duration": 1.0
```
Comment (chrislo27) [inline core/assets/sounds/cues/clappyTrio/data.json]: Indentation

File: core/assets/sounds/cues/clappyTrio/data.json [hunk 4]
```diff
@@ -58,7 +59,8 @@
         },
         {
           "id": "clappyTrio/clap",
-          "beat": 3.0
+          "beat": 3.0,
+		  "duration": 1.0
```
Comment (chrislo27) [inline core/assets/sounds/cues/clappyTrio/data.json]: Indentation

File: core/assets/sounds/cues/clappyTrioGba/data.json [hunk 5]
```diff
@@ -30,7 +30,8 @@
         },
         {
           "id": "clappyTrioGba/clap",
-          "beat": 2.0
+          "beat": 2.0,
+		  "duration": 1.0
```
Comment (chrislo27) [inline core/assets/sounds/cues/clappyTrioGba/data.json]: Indentation

File: core/assets/sounds/cues/clappyTrioGba/data.json [hunk 6]
```diff
@@ -53,7 +54,8 @@
         },
         {
           "id": "clappyTrioGba/clap",
-          "beat": 3.0
+          "beat": 3.0,
+		  "duration": 1.0
```
Comment (chrislo27) [inline core/assets/sounds/cues/clappyTrioGba/data.json]: Indentation

**Review by chrislo27 — CHANGES_REQUESTED** [review]
Your commits still contain "Fix #179".

Do not ever use words in commits that may lead to GitHub closing issues (fixed, closed, completed, etc).

I'm letting you off this time. It shouldn't take three pull requests to fix this simple change.

File: core/assets/sounds/cues/clappyTrioGba/data.json (hunk 5 above)
Comment (GuardedLolz) [inline core/assets/sounds/cues/clappyTrioGba/data.json]: wut

Comment (GuardedLolz) [reply core/assets/sounds/cues/clappyTrioGba/data.json]: Wut

File: core/assets/sounds/cues/clappyTrioGba/data.json (hunk 6 above)
Comment (GuardedLolz) [reply core/assets/sounds/cues/clappyTrioGba/data.json]: I mean this is literally the changes I've made with a subtle clue.

File: core/assets/sounds/cues/builtToScaleDS/data.json (hunk 1 above)
Comment (GuardedLolz) [inline core/assets/sounds/cues/builtToScaleDS/data.json]: wut

**Pull request merged**